react/features/mobile/navigation/components/welcome/functions.js

```diff
@@ -18,7 +17,6 @@ import { navigationStyles } from '../styles';
 export function renderArrowBackButton(onPress: Function) {
     return (
         <HeaderNavigationButton
-            buttonStyle = { navigationStyles.arrowBackStyle }
```

Comment:
Why is this no longer necessary?

Reply:
Based on visual testing, this is no longer necessary.